scm/etc/scripts/UFS_case_gen.py

```diff
@@ -3051,7 +3051,7 @@ def write_SCM_case_file(state, surface, oro, forcing, init, case, date, forcing_
     com = 'mkdir -p ' + PROCESSED_CASE_DIR
     print(com)
     os.system(com)
-    fileOUT = os.path.join(PROCESSED_CASE_DIR, case + '_fm_{}'.format(forcing_method) + ('_vm_{}'.format(vertical_method) if forcing_method == 2 else '') + ('_geos' if geos_wind_forcing else '') + ('_wind_nudge' if wind_nudge else '') + '_SCM_driver.nc')
```

Comment:
@grantfirl This was causing some problems so I removed it. We can add it back in, but it requires more changes to work.

Reply:
No problem. I added this to make it easier to see what forcing a file uses without have to ncdump it to see the global attributes.